packages/docsearch-react/src/useDocSearchKeyboardEvents.ts

```diff
@@ -39,7 +39,8 @@ export function useDocSearchKeyboardEvents({
       if (
         (event.keyCode === 27 && isOpen) ||
         // The `Cmd+K` shortcut both opens and closes the modal.
-        (event.key.toLowerCase() === 'k' && (event.metaKey || event.ctrlKey)) ||
+        (event.key?.toLowerCase() === 'k' &&
```

Comment:
@powerfulyang Can you add a comment saying that there's a bug in Chrome where `event.key` can be `undefined`, and link to https://github.com/paperjs/paper.js/issues/1398?

Reply:
[scrnli_2023_5_16 20-27-03.webm](https://github.com/algolia/docsearch/assets/21997724/c64dc193-d88b-4d2d-ab95-ef1f45103d9c)
I'm sorry, it's my fault. Watching this video may make things clearer.